doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -395,6 +395,7 @@ Other samples
 
     * Added new configuration that builds the sample with support for remote IPC Service shell on nRF5340 application core through USB.
     * Added possibility to build with the limited nRF21540 front-end module hardware pinout.
+    * Improved total payload size calculation for radio duty cycle.
```

Comment:
I might want to twist this a bit because this has a long chain of nouns put together like "total payload size calculation". Maybe something like "Improved the calculation of the total payload size for the radio duty cycle." Not a big deal, but might make it a wee clearer.

Reply:
Fixed and added information about ramp-up and duty cycle limitation.